src/aap_eda/api/views/decision_environment.py

```diff
@@ -117,3 +109,26 @@ def retrieve(self, request, pk):
                 decision_environment.data
             ).data
         )
+
+    @extend_schema(
```

Comment:
new param must be added to the openapi spec. 
```suggestion
    @extend_schema(
        parameters=[
            OpenApiParameter(
                name='force',
                description='Force deletion even if there are dependent objects',
                required=False,
                type='boolean',
            )
        ],
```

Reply:
added.